src/components/Login.tsx

```diff
@@ -2,7 +2,7 @@ import styled from 'styled-components/macro'
 import heroImg from '../assets/images/Login-hero.jpg'
 import Button from './common/Button/Button'
 
-const Login = (): JSX.Element => {
+function Login(): JSX.Element {
   const authEndpoint: string = 'https://accounts.spotify.com/authorize'
   const redirectUri: string = process.env.REACT_APP_REDIRECT_URI as string
```

Comment:
Isn't that a bit too much type checking? Type inference should take care of the first type in this line. Doesn't it?

Reply:
Indeed it would. In order to be consistent I tried to be explicit wherever possible. Do you have an opinion about that?

Reply:
I'd say don't add code, if it is already type safe by inference. That's just visual noise.